### PR TITLE
Add Module : WordPress Plugins detect

### PR DESCRIPTION
**Pull request by OussamaBeng**
Adding module to detect name and version of the installed plugins in WordPress websites

### CODE REVIEW

**Review by fwininger — APPROVED** [review]
Seems a very great feature ! Thanks

**Comment by fwininger** [comment]
@jinxka take a look

**Review by bretfourbe — APPROVED** [review]
It would be nice to add tests for this module

File: wapitiCore/attack/mod_cms.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+from os.path import join as path_join
+
+from wapitiCore.net.web import Request
+from wapitiCore.attack.attack import Attack
+from wapitiCore.language.vulnerability import LOW_LEVEL, _
+from wapitiCore.definitions.fingerprint import NAME as TECHNO_DETECTED
+
+MSG_TECHNO_VERSIONED = _("{0} {1} detected")
+MSG_NO_WP = _("No WordPress Detected")
+class mod_cms(Attack):
```
Comment (devl00p) [inline wapitiCore/attack/mod_cms.py:12]: If it is specific to Wordpress you should rename to `mod_wordpress_plugins` or `mod_wordpress_enumeration` so it is more clear

File: wapitiCore/attack/mod_cms.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    def get_plugin(self):
+        with open(path_join(self.DATA_DIR, self.PAYLOADS_FILE), errors="ignore") as plugin_list:
+            for line in plugin_list:
+                plugin= line.strip()
+                if plugin:
+                    yield plugin
+
+    def detect_plugin(self, url):
+        for plugin in self.get_plugin():
+            req= Request('{}/wp-content/plugins/{}/readme.txt'.format(url, plugin))
```
Comment (devl00p) [inline wapitiCore/attack/mod_cms.py:26]: Can you please fix whitespaces around operators to comply with PEP8 recommandations ? https://www.python.org/dev/peps/pep-0008/#other-recommendations

**Review by devl00p — CHANGES_REQUESTED** [review]

File: wapitiCore/attack/mod_cms.py (hunk 1 above)
Comment (devl00p) [reply wapitiCore/attack/mod_cms.py:12]: `mod_wpenum` would be short and meaningful I guess

**Review by bretfourbe — APPROVED** [review]

**Comment by devl00p** [comment]
Nice, than for your work

**Pull request merged**